### PR TITLE
Update disabled state of chameleon checkbox#174

### PR DESCRIPTION
**Pull request by jbforth4**

### CODE REVIEW

File: packages/checkbox/src/chameleon-checkbox-style.ts [hunk 1]
```diff
@@ -30,6 +30,11 @@ export default css`
     width: 20px;
   }
 
+  /* .horizontal-line {
+    border: 10px solid green;
+    border-radius: 5px;
+  } */
+
```
Comment (ryuhhnn) [inline packages/checkbox/src/chameleon-checkbox-style.ts]: Can you remove this?

Comment (jbforth4) [reply packages/checkbox/src/chameleon-checkbox-style.ts]: Whoops, sure thing.

**Comment by jbforth4** [comment]
what should the version be? 1.2.0?

**Comment by tfreiner** [comment]
> what should the version be? 1.2.0?

yep!

**Pull request merged**